README.md

```diff
@@ -251,7 +251,7 @@ services.AddPullDeliverySubscription()
 
 // Method 2: Register all domain event handlers from a given assembly
 services.AddPullDeliverySubscription()
-    .AddDomainEventHandlers(Assembly.GetExecutingAssembly());
+    .AddDomainEventHandlers(ApplicationAssemblyHandle.Assembly);
```

Comment:
2 things: 
- If we're going to change this, then it would also need to be changed at line 148
- But I would keep it the way it was before since we don't show what `ApplicationAssemblyHandle` is

Reply:
That's fair, the reasoning for this change is that ApplicationAssemblyHandle was used for LeapShop and I guess for most users, we can ask them to use executingassembly instead.